### ISSUE TITLE
Incorrect events dispatched randomly

### ISSUE THREAD

**Issue by aacook** [mikechambers/Simple-Game-Framework#1]
Hey Mike,

Thanks for putting this together.

In GestureManager I'm experiencing a problem with it dispatching the wrong events.  I'll be swiping up and it will work correctly for a while, then I'll try changing it to swiping down, and it will continue to think it's an up gesture.  Every once and a while it thinks the down gesture is in fact a down gesture, but then it will go back to being an up gesture...

I don't really have any specific way to reproduce.

**Comment by mikechambers** [comment]
Yeah. I have gotten some other reports of this and will look into it.

Are you testing on a device, or on the desktop?

If anyone else can reproduce, please let me know.

**Comment by aacook** [comment]
Thanks for the reply!

I tested first just on my device.  Then I realized you can test on your desktop by making the gesture with your mouse, which is great.

It mainly happens when I switch x/y directions, but sometimes I'll be swiping up and it will think I'm swiping down.

I removed all of my tweening and I'm just tracing out the events, and it appears to be dispatching the wrong events.

This is great though, thanks for putting it together.

Alex
